CHANGELOG.md

```diff
@@ -15,6 +19,12 @@
   by the ratelimiter - Github issue #1439
 - Invalid fields in rate limiter related API requests are now failing with
```

Comment:
nit: This is also related to a issue #1437.

Reply:
I agree we should add the issue numbers where applicable (like here). Also +1 on consistent style when referencing issue numbers.

Reply:
Fixed.